data/compatible_products.yml

```diff
@@ -95,14 +96,6 @@ sets:
       - version: 5.1.0
         compatibility: 'https://eclipse-ee4j.github.io/glassfish/certifications/jakarta-web-profile/8.0/TCK-Results'
 
-    - name: Open Liberty
-      vendor: IBM Corporation
-      image: '/images/compatible_products/ibm_open_liberty.png'
-      download: 'https://openliberty.io/downloads/'
-      versions:
-      - version: 19.0.0.6
```

Comment:
We shouldn't be deleting the old version (19.0.0.6), just adding the new version (20.0.0.3).  This was the whole reason we reverted the original PR for this and worked through the website updates to allow for multiple versions.